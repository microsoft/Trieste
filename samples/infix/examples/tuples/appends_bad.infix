print "no args" append;
print "duplicate" append append((1, 2));
print "arithmetic" append + append;