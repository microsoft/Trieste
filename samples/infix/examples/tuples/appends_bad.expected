//! --enable-tuples --tuples-require-parens expect_parse_fail
//! --enable-tuples --tuples-require-parens --use-parser-tuples expect_parse_fail
//! --enable-tuples expect_parse_fail
Invalid append
-- appends_bad.infix:0:16
print "no args" append;
                ~~~~~~

Invalid append
-- appends_bad.infix:1:18
print "duplicate" append append((1, 2));
                  ~~~~~~

Invalid append
-- appends_bad.infix:2:19
print "arithmetic" append + append;
                   ~~~~~~

Invalid append
-- appends_bad.infix:2:28
print "arithmetic" append + append;
                            ~~~~~~
